src/map/spell.cpp

```diff
@@ -452,7 +424,7 @@ namespace spell
     void LoadSpellList()
     {
         const char* Query = "SELECT spellid, name, jobs, `group`, family, validTargets, skill, castTime, recastTime, animation, animationTime, mpCost, \
-                             AOE, base, element, zonemisc, multiplier, message, magicBurstMessage, CE, VE, requirements, content_tag, spell_range \
+                             AOE, base, element, zonemisc, multiplier, message, magicBurstMessage, CE, VE, requirements, content_tag, type, spell_range \
```

Comment:
While we are here, why dont we order this properly, following the same order as the database?

Reply:
Sounds good. I'll make it happen.

Reply:
RIP `requirements`
```sql
INSERT INTO `spell_list` VALUES (16,'blindna',@G_WHITE,@F_NA,@E_LIGHT,@VT_SUPPSPL,@S_HEALING,@T_HMNA,0,16,1000,10000,0,0,16,2000,4,0,1.00,1,165,4,204,@CT_BASE,'WHM:14,SCH:17:AW');
```

Reply:
I will say, if this grows to big, it will be a nightmare to review properly.

Maybe the simpler changes should go in separate PRs first.

Reply:
There won't be anything difficult to review by the time I'm done. I already reordered everything in the SQL query, and arranged everything that comes after before starting this change. It won't take me much to rework the logic required to read the database.
```sql
INSERT INTO `spell_list` VALUES (11,'curaga_v',@G_WHITE,@F_CURAGA,@E_LIGHT,@VT_SLF_PRTY,@S_HEALING,@T_BUFFHMCURE,0,380,5750,11000,0,0,11,2000,1,0,1.00,0,0,204,@CT_ABY,'WHM:91');
INSERT INTO `spell_list` VALUES (12,'raise',@G_WHITE,@F_RAISE,@E_LIGHT,@VT_PDEAD,@S_HEALING,@T_RAISE,0,150,15000,60000,0,0,12,2000,0,0,1.00,0,0,204,@CT_BASE,'WHM:25,RDM:38,PLD:50,SCH:35');
INSERT INTO `spell_list` VALUES (13,'raise_ii',@G_WHITE,@F_RAISE,@E_LIGHT,@VT_PDEAD,@S_HEALING,@T_RAISE,0,150,14000,60000,0,0,13,2000,0,0,1.00,0,0,204,@CT_BASE,'WHM:56,RDM:95,SCH:70:AW'); -- req 4
```

Reply:
If it makes you feel any better about the scope of this, I'm pulling all the job and level data off of the scrolls at the auction house to put in the new col.
```sql
INSERT INTO `spell_list` VALUES (152,'blizzard_iv',@G_BLACK,@F_BLIZZARD,@E_ICE,@VT_ENEMY,@S_ELEMENTAL,@T_NONE,0,162,6000,30000,2,252,152,2000,0,506,2.00,0,0,204,@CT_BASE,'BLM:74,RDM:89,SCH:74:AB,GEO:88'); -- req 2
INSERT INTO `spell_list` VALUES (153,'blizzard_v',@G_BLACK,@F_BLIZZARD,@E_ICE,@VT_ENEMY,@S_ELEMENTAL,@T_NONE,0,267,10000,45000,2,252,153,2000,0,829,2.30,0,0,204,@CT_ABY,'BLM:89,SCH:95:AB'); -- req 2
INSERT INTO `spell_list` VALUES (154,'aero',@G_BLACK,@F_AERO,@E_WIND,@VT_ENEMY,@S_ELEMENTAL,@T_NONE,0,6,500,2000,2,252,154,2000,0,25,1.00,0,0,204,@CT_BASE,'BLM:9,RDM:14,DRK:17,SCH:12,GEO:14');
```
for example
req 1 looks like merit ability from what I can see, could easily be "WHM:75:M"
req 2 is Addendum: Black "SCH:99:AB"
req 4 is Addendum: White "SCH:99:AW"
req 8 is Tabula Rasa, and I have that for Embrava now "SCH:5:TR"
req 16 is for the latest blu stuff
req 32 is for geo stuff